Common/src/toolboxes/MMS/CMMSNSTwoHalfSpheresSolution.cpp

```diff
@@ -391,6 +391,6 @@ void CMMSNSTwoHalfSpheresSolution::GetMMSSourceTerm(const su2double *val_coords,
   val_source[4] /= Velocity_Ref*Pressure_Ref;
 }
 
-bool CMMSNSTwoHalfSpheresSolution::IsManufacturedSolution(void) {
+bool CMMSNSTwoHalfSpheresSolution::IsManufacturedSolution(void) const {
```

Comment:
@economon, @vdweide, I put the const and virtual specifiers on the MMS classes and striped the trailing spaces